api/SVGAnimationElement.json

```
@@ -622,6 +628,7 @@
         "__compat": {
           "description": "<code>repeatEvent</code> event",
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/SVGAnimationElement/repeatEvent_event",
+          "spec_url": "https://svgwg.org/svg2-draft/single-page.html#interact-RepeatEvent",
```

We should decide if we go for the single-page.html or not. Given the other spec urls in this PR I guess the preference is the multi page variant.

```suggestion
          "spec_url": "https://svgwg.org/svg2-draft/interact.html#RepeatEvent",
```